shell/browser/api/electron_api_browser_window.cc

```diff
@@ -362,7 +362,11 @@ void BrowserWindow::Blur() {
 
 void BrowserWindow::SetBackgroundColor(const std::string& color_name) {
   BaseWindow::SetBackgroundColor(color_name);
-  web_contents()->SetPageBaseBackgroundColor(ParseHexColor(color_name));
+  SkColor color = ParseHexColor(color_name);
+  web_contents()->SetPageBaseBackgroundColor(color);
+  auto* rwhv = web_contents()->GetRenderWidgetHostView();
+  if (rwhv)
+    rwhv->SetBackgroundColor(color);
```

Comment:
If we add the `WebContents::SetBackgroundColor()` method, this function could become
```cpp
void BrowserWindow::SetBackgroundColor(const std::string& color_name) {
  BaseWindow::SetBackgroundColor(color_name);
  if (api_web_contents_) {
    api_web_contents_->SetBackgroundColor(ParseHexColor(color_name));
  }
}
```